cime_config/usermods_dirs/BmidH/#user_nl_clm#

```diff
@@ -0,0 +1,107 @@
+!----------------------------------------------------------------------------------
```

Comment:
It looks like this file was accidentally added. It should be removed.

Reply:
Removed